tests/integ/datasource/test_databricks.py

```diff
@@ -238,3 +239,23 @@ def test_unit_data_source_data_to_pandas_df():
     assert df.to_dict(orient="records") == [
         {"COL1": 1, "COL2": '{"key1": "value1", "key2": "value2"}'}
     ]
+
+
+def test_unicode_column_databricks(session):
+    df = session.read.dbapi(create_databricks_connection, table="User_profile_unicode")
+    Utils.check_answer(df, [Row(编号=1, 姓名="山田太郎", 国家="日本", 备注="これはUnicodeテストです")])
+
+
+def test_double_quoted_column_databricks(session):
+    df = session.read.dbapi(create_databricks_connection, table="User_profile")
+    Utils.check_answer(
+        df,
+        [
+            Row(
+                id=1,
+                name="Yamada Taro",
+                country="Japan",
+                remarks="This is a test remark",
```

Comment:
does `Utils.check_answer` check column name? I thought it's only checking the value ignoring the column name